core/pubkeyConverter/hexPubkeyConverter_test.go

```diff
@@ -83,7 +83,28 @@ func TestHexPubkeyConverter_EncodeDecodeShouldWork(t *testing.T) {
 	buff, err := hpc.Decode(value)
 	assert.Nil(t, err)
 
-	revoveredValue := hpc.Encode(buff)
+	recoveredValue, _ := hpc.Encode(buff)
```

Comment:
please also test that the returned err is nil

Reply:
done